### PR TITLE
Standardize strings

### PR DESCRIPTION
**Pull request by danielbreves**
:koala:

Seems like we have been editing en.yml directly.

/cc @zendesk/quokka @ydrain @pdeuter 

### Risks
 - None

### CODE REVIEW

**Comment by sandlerr** [comment]
:+1:

**Comment by sandlerr** [comment]
maybe change the comment to `# This file is generated by rake i18n:standardize. Do not edit it directly`

**Comment by xuebinglee** [comment]
:+1:

**Comment by xuebinglee** [comment]
Why not make the comment IN CAPITAL LETTERS SO THAT IT LOOKS SCARY? :imp:

**Comment by danielbreves** [comment]
I think a test would be better, but not sure it's worth it right now.

**Comment by sandlerr** [comment]
you need to re-run it with the new header :-)

**Comment by danielbreves** [comment]
@sandlerr :haircut:

**Comment by sandlerr** [comment]
LGTM!

File: config/locales/translations/zendesk_apps_support.yml [hunk 1]
```diff
@@ -100,13 +108,17 @@ parts:
...
       value: "Missing translation file for locale '%{defaultLocale}'. Please read: http://developer.zendesk.com/documentation/apps/i18n.html"
   - translation:
+      key: "txt.apps.admin.error.app_build.invalid_host"
+      title: "App builder job: invalid host application for apps"
+      value: "%{host_name} is an invalid host for apps."
+  - translation:
       key: "txt.apps.admin.error.app_build.invalid_location.one"
       title: "App builder job: invalid locations"
-      value: "%{invalid_locations} is an invalid location."
+      value: "%{invalid_locations} is an invalid location in %{host_name}."
```
Comment (ydrain) [inline config/locales/translations/zendesk_apps_support.yml]: Please provide the example values of `%{invalid_locations}` & `%{host_name}` for translators in `title`. Thanks!

File: config/locales/translations/zendesk_apps_support.yml [hunk 2]
```diff
@@ -100,13 +108,17 @@ parts:
...
   - translation:
       key: "txt.apps.admin.error.app_build.invalid_location.other"
       title: "App builder job: invalid locations"
-      value: "%{invalid_locations} are invalid locations."
+      value: "%{invalid_locations} are invalid locations in %{host_name}."
```
Comment (ydrain) [inline config/locales/translations/zendesk_apps_support.yml]: This one too.

**Pull request merged**